features/frontend/products.feature

```diff
@@ -71,3 +84,36 @@ Feature: Products
     Scenario: Display product attributes
         Given I am on the product page for "Super T-shirt"
         Then I should see "Wool"
+
+    Scenario: Receiving exception while entering page for product with empty slug
+        Given I go to page for product with empty slug
+         Then the response status code should be 404
+
+    Scenario: Displaying product rating and reviews:
+        Given I am on the store homepage
+        And I follow "PHP T-Shirts"
+        When I click "Symfony T-Shirt"
+        Then I should see "Rating: 5"
+        And I should see 1 review on the reviews list
```

Comment:
http://forum.wordreference.com/threads/in-the-list-or-on-the-list.469925/

Pretty sure it should be **on** ;)